tests/test_db_pool.py

```
@@ -0,0 +1,96 @@
+    mock_session.rollback.return_value.set_result(None)
+    
+    # Mock async context manager: __aenter__ returns session, __aexit__ returns None (awaitable)
+    mock_session.__aenter__ = AsyncMock(return_value=mock_session)
+    mock_session.__aexit__ = AsyncMock(return_value=None)
+
+    # Patch the async_session_maker in the RELOADED module
+    with patch.object(mock_db_module, "async_session_maker", return_value=mock_session) as mock_maker:
+        
+        async def task():
+            # Use the get_db from the RELOADED module
+            async for session in mock_db_module.get_db():
```

Ruff (0.14.14)

[warning] Loop control variable `session` not used within loop body

Rename unused `session` to `_session`

(B007)